src/components/Balance.test.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { render, screen } from '../testUtils'
+import { BTC, SATS } from '../utils'
+
+import Balance from './Balance'
+
+describe('<Balance />', () => {
+  it('should hide balance for BTC by default', () => {
+    render(<Balance value={'123.456'} unit={BTC} />)
+    expect(screen.getByText(`*****`)).toBeInTheDocument()
+    expect(screen.queryByText(`123.45 600 000`)).not.toBeInTheDocument()
+  })
+
+  it('should hide balance for SATS by default', () => {
+    render(<Balance value={'123'} unit={SATS} />)
+    expect(screen.getByText(`*****`)).toBeInTheDocument()
+    expect(screen.queryByText(`123`)).not.toBeInTheDocument()
+  })
+
+  it('should render a string BTC value correctly as BTC', () => {
+    render(<Balance value={'123.03224961'} unit={BTC} showBalance={true} />)
+    expect(screen.getByText(`123.03 224 961`)).toBeInTheDocument()
```

Comment:
```suggestion
    expect(screen.getByText(`123.03 224 961`)).toBeInTheDocument()
    expect(screen.get(`₿`)).toBeInTheDocument()
```
Should we test the currency sign as well?

Reply:
Left it out since that would only work for BTC but not for sats (svg).